apps/bundle-size/src/commands/build.js

```diff
@@ -0,0 +1,67 @@
+// @ts-check
+
+const chalk = require('chalk');
+const Table = require('cli-table3');
+const fs = require('fs-extra');
+const glob = require('glob');
+const path = require('path');
+const prettyBytes = require('pretty-bytes');
+
+const buildFixture = require('../utils/buildFixture');
+const { hrToSeconds } = require('../utils/helpers');
+const prepareFixture = require('../utils/prepareFixture');
+
+/**
+ * @param {{ verbose: boolean }} options
```

Comment:
Is there really that much console output from this command ? and does it make sense to not print out results to console by default ?

IMO we don't really need `verbose` and if it's a problem for CI, rather have a `silent` flag

Reply:
> IMO we don't really need `verbose` and if it's a problem for CI, rather have a `silent` flag

👍

Reply:
I decided to go with `--quiet` as `start-storybook` has the same flag, in 51b8acb